configs/build_config.py

```diff
@@ -127,15 +126,23 @@ def __init__(self, flask_config_path, autoratify=True):
                 raise ValueError('\n'.join(errors))
 
 
-    def get(self, key, default_value):
+    def get(self, key, default_value=None):
```

Comment:
I'm trying to remember why this is here along with the implicit __getitem__ at line 142.  It should be amended per the comments there and can we get rid of this get()?

Reply:
This allows a dictionary-like behavior to get property with default value. A standard class.__get__("Property_name") works, but there is no __get__ with default return in standard class properties, as far as I know.

Reply:
No there's not but you can override the descriptor protocol (data model) methods and have it return None if the key doesn't exist.